### DISCUSSION TITLE
Amazon.Lambda.Logging.AspNetCore - Always logs level as "trce"

### DISCUSSION THREAD

**Discussion by bradyjp1974** [aws/aws-lambda-dotnet#1889] [Q&A] [answered]
Using AddLambdaLogger I've added the Amazon.Lambda.Logging.AspNetCore v3.1.1 provider to my Lamba logging capabilities.

One issue I am experiencing is that no matter what logging statement I use, the logs are always written with a level of 'trce' in CloudWatch.

If I enable the "IncludeLogLevel" LambdaLoggerOptions then the correct log level is included inside the log message.

These are my logging statements and the logging outcomes are show in the screenshot below.

        logger.LogInformation("ILogger LogInformation");
        logger.LogWarning("ILogger LogWarning");
        logger.LogError("ILogger LogError");
        logger.LogCritical("ILogger LogCritical");

![sample](https://github.com/user-attachments/assets/38886f6f-d607-4eaf-b76f-9f621c9d51c5)

Can anyone confirm if I am doing something stupid here, or is this the expected behaviour?

Thank you!

**Comment by normj** [comment, marked as answer]
Hi @bradyjp1974 this is an issue with our libraries and how they interact with ASP.NET Core logging. I'm working on a fix and hope to push out a PR soonish.

**Reply by bradyjp1974** [reply]
Thank you for confirming, much appreciated.
